packaging/flatpak/mirage.flatpak.yaml

```diff
@@ -560,39 +542,57 @@ modules:
 - name: python3-python-olm
   buildsystem: simple
   build-commands:
-  - pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST}
-    python-olm==3.1.3
+  - pip3 install --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST}
+    "python-olm==3.1.3"
   sources:
-  - type: file
-    url: https://files.pythonhosted.org/packages/0f/86/e19659527668d70be91d0369aeaa055b4eb396b0f387a4f92293a20035bd/pycparser-2.20.tar.gz
-    sha256: 2d475327684562c3a96cc71adf7dc8c4f0565175cf86b6d7a404ff4c771f15f0
-  - type: file
-    url: https://files.pythonhosted.org/packages/05/54/3324b0c46340c31b909fcec598696aaec7ddc8c18a63f2db352562d3354c/cffi-1.14.0.tar.gz
-    sha256: 2d384f4a127a15ba701207f7639d94106693b6cd64173d6c8988e2c25f3ac2b6
   - type: file
     url: https://files.pythonhosted.org/packages/ae/e7/d9c3a176ca4b02024debf82342dab36efadfc5776f9c8db077e8f6e71821/pycparser-2.20-py2.py3-none-any.whl
     sha256: 7582ad22678f0fcd81102833f60ef8d0e57288b6b5fb00323d101be910e35705
   - type: file
     url: https://files.pythonhosted.org/packages/45/0b/38b06fd9b92dc2b68d58b75f900e97884c45bedd2ff83203d933cf5851c9/future-0.18.2.tar.gz
     sha256: b1bead90b70cf6ec3f0710ae53a525360fa360d306a86583adc6bf83a4db537d
+  - type: file
+    url: https://files.pythonhosted.org/packages/05/54/3324b0c46340c31b909fcec598696aaec7ddc8c18a63f2db352562d3354c/cffi-1.14.0.tar.gz
+    sha256: 2d384f4a127a15ba701207f7639d94106693b6cd64173d6c8988e2c25f3ac2b6
   - type: file
     url: https://files.pythonhosted.org/packages/d4/a4/1face47e65118d7c52726dfa305410a96bc4a0c6f3f99c90bc7104aebf21/python-olm-3.1.3.tar.gz
     sha256: 9a6c6133ce3777788c88e3f18b13c5b36a2f76ed1a0e774d1a48adf542fee871
+- name: python3-PyYAML
+  buildsystem: simple
+  build-commands:
+  - pip3 install --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST}
+    "PyYAML==5.3.1"
+  sources:
+  - type: file
+    url: https://files.pythonhosted.org/packages/64/c2/b80047c7ac2478f9501676c988a5411ed5572f35d1beff9cae07d321512c/PyYAML-5.3.1.tar.gz
+    sha256: b8eac752c5e14d3eca0e6dd9199cd627518cb5ec06add0de9d32baeee6fe645d
+- name: python3-requirements-parser
+  buildsystem: simple
+  build-commands:
+  - pip3 install --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST}
+    "requirements-parser==0.2.0"
+  sources:
+  - type: file
+    url: https://files.pythonhosted.org/packages/03/80/eb6ba1dd0429089436e90e556db50884ea21da060b10f2e5668c4cac99da/requirements-parser-0.2.0.tar.gz
+    sha256: 5963ee895c2d05ae9f58d3fc641082fb38021618979d6a152b6b1398bd7d4ed4
```

Comment:
These dependencies used for the generation (PyYAML and requirements-parser) shouldn't end up in the final manifest, I don't get them when I generate it on my machine. Could you investigate to find which step in the scripts is responsible?

Reply:
It is a mess with these dependencies generation. Right now, if I use fully automatic generation, it will not compile for me (few packages get messed up with setuptools). So, I had to restore those packages manually to prev version.

As for YAML and req parser, they are not there if we remove venv before rebuild. 

Updated manifest submitted via forced commit, please review.